README.md

```diff
@@ -8,6 +8,8 @@
 
 Loggly broadcasting for Good.
 
+`good-loggly` is a [good](https://github.com/hapijs/good) reporter implementation to write [hapi](http://hapijs.com/) server events to Loggly.
```

Comment:
We use it with good because we change the logging methods based on our environment. So I don't have an example without configuring good.